contractTest/gateways/MsSQL/retrieveDepartmentByCode.contractTest.js

```diff
@@ -0,0 +1,44 @@
+import retrieveDepartmentByCodeGateway from "../../../src/gateways/MsSQL/retrieveDepartmentByCode";
+import { setupOrganisationFacilityDepartmentAndManager } from "../../../test/testUtils/factories";
+import AppContainer from "../../../src/containers/AppContainer";
+
+describe("retrieveActiveDepartmentsByFacilityIdGateway", () => {
+  const container = AppContainer.getInstance();
+  it("retrieves department returning object containing trustId, wardId and wardCode", async () => {
+    // Arrange
+    const email = `${Math.random()}@nhs.co.uk`;
```

Comment:
You won't need to do this once https://github.com/madetech/nhs-virtual-visit/pull/803 is merged, `setupOrganisationFacilityDepartmentAndManager()` will be able to create the Manager fine without the email needing to be specified because the database will be cleared between tests.

Reply:
Yes, this is good news! I will wait until #803 is merged.

Reply:
This has been refactored for all previous contract test as well.